src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -204,10 +209,12 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
               isSubmitting={isSubmitting}
               register={register}
               watch={watch}
-              {...rest}
-              onSubmit={onFormSubmit}
+              onSubmit={onSubmit}
               editing={!!venueId}
               formError={formError}
+              control={control}
+              handleSubmit={handleSubmit}
+              errors={errors}
             />
```

Comment:
Now that you have removed `...rest` at all, are we sure that we pass every needed prop there? Did you go one-by-one to check what `rest` object contained?

Reply:
I didn't check the rest object, I just checked the expected Props in the component and passed only the ones being used/necessary.

Reply:
Is it possible that rest had some `optional` props that were used and now we are not passing them?

Reply:
Nope, I checked the interface of the component and every prop is being passed. If the `rest` object had any other fields that means they were not being used by the component.